<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>服务器地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serverIPcomboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>124.243.202.5</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverPortlineEdit">
       <property name="text">
        <string>18901</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sCleanpushButton">
       <property name="text">
        <string>清  空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="servertextBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="statuslabel">
       <property name="text">
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="listenpushButton">
       <property name="text">
        <string>侦  听</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stoppushButton">
       <property name="text">
        <string>停  止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="serverMessagelineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="serverSendpushButton">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
